### PR TITLE
Fix "Save As" overflow on small screens

### LINKED ISSUE #789: Improving the save as button in smaller screens.
### Dropdown of the save as png option is not good in smaller screens.

> We can make the size of the button to medium size

>
screenshot from 2019-02-18 12-58-07

It looks like this  in smaller screens.

### Where to look.

http://sequencer.publiclab.org

### What's your PublicLab.org username?
harshithpabbati

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by lohitha02**
Fixes #789

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by lohitha02** [comment]
#789

**Comment by Harshithpabbati** [comment]
@lohitha02 mention @publiclab/is-reviewers to make it review :)

**Comment by HarshKhandeparkar** [comment]
@lohitha02 please link the issue in the PR body with the keyword Fixes as this will close the issue as soon as the PR is merged. Thanks!

**Comment by HarshKhandeparkar** [comment]
Next time please ask in the issue discussion if you can accept the issue. This will make sure that no two contributors will work on the same issue. Thanks

File: examples/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+z<!DOCTYPE html>
```
Comment (HarshKhandeparkar) [inline examples/index.html]: Please remove this

**Comment by HarshKhandeparkar** [comment]
You can descriptively title the PR to make it easier for the reviewers to review it. Thanks!

**Comment by Harshithpabbati** [comment]
@lohitha02 jst make the changes said by @HarshKhandeparkar and push it again.
Thanks:)

**Comment by Harshithpabbati** [comment]
@lohitha02  thanks for your contributions and we need more contributions from you :+1:

**Comment by lohitha02** [comment]
publiclab

**Comment by lohitha02** [comment]
@publiclab/reviewers please review this.

**Comment by Harshithpabbati** [comment]
@lohitha02 please mention @publiclab/is-reviewers 
Thanks:)

**Comment by jywarren** [comment]
This is super. Thank you so much, and thanks everyone for the helpful comments!

**Pull request merged**